ci/test/pipeline.template.yml

```diff
@@ -137,7 +137,7 @@ steps:
         inputs:
           - "*"
         depends_on: []
-        timeout_in_minutes: 10
+        timeout_in_minutes: 20
```

Comment:
Unrelated (and outdated?) change

Reply:
Here this hung for 10 minutes just doing the docker pull, so I think we still want this. (but yes, unrelated)

Reply:
Ok